src/shims/unix/linux/epoll.rs

```diff
@@ -50,6 +59,15 @@ pub struct EpollEvent {
     pub ready_list: Rc<RefCell<BTreeMap<(WeakFileDescriptor, i32), EpollReturn>>>,
 }
 
+impl EpollEvent {
+    pub fn contains_flag(&self, flag: u32) -> bool {
+        if self.events & flag == flag {
```

Comment:
```suggestion
        self.events & flag == flag
```

Just make this the body of the function